scripts/check-gomod-version.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [[ $2 = "-v" ]]; then
+    v () {
+        >&2 echo "$@"
```

Comment:
you didn't want to just `set -x`?

Reply:
I think I was initially thinking that 🤔
lemme check.  if it shows the output, yea, I'll just `-x` it.  it's even better for troubleshooting, and it's not a ton of output.

Reply:
yep, `-x` ftw.  thanks!